api/handler/seattle.go

```diff
@@ -127,10 +127,10 @@ func (h *Handler) seattleGetOfficerByBadgeHistorical(badge string, w http.Respon
 	}
 
 	sort.Slice(officers, func(a, b int) bool {
-		if officers[a].LastName == officers[b].LastName {
-			return officers[a].FirstName < officers[b].FirstName
+		if officers[a].Date == officers[b].Date {
+			return officers[a].Date > officers[b].Date
```

Comment:
If both officers have the same date, shouldn't we have another tiebreaker for sorting?

Reply:
There would be only one officer returned in this function. Or there should be anyway. If there's more than one officer returned then something else has gone very wrong.

Reply:
right, this is get by badge. If thats the case, do we need the sorting function at all?

Reply:
This is for historical look ups so it's to sort the returned roster entries by their associated roster date.

Reply:
On Barry's note though, isn't that sorting something we can do in the DB query so we don't have to do it here?

Reply:
I think the db is already doing an `ORDER BY o.date DESC`

https://github.com/SeattleDSA/spd-lookup/blob/9e706b2d3407eeb11940558f557f545e50e91c7f/api/data/seattle_db.go#L156-L171

Reply:
Oop, yeah that's true huh. I'll just go ahead take that part out.